packages/code-block/partials/clipboard-button/style.module.css

```
@@ -1,42 +1,84 @@
-/*
-Note: Expects CSS properties from 
-code-block/theme-(dark|light).module.css
-to be present.
-*/
-
 .button {
-  composes: g-type-body-small-strong from global;
+  /* variables for math */
+  --border-width: 1px;
+  --padding-top-bottom: calc(6px - var(--border-width));
+  --padding-left-right: calc(12px - var(--border-width));
+
+  /* variables from design-tokens repo */
+  --token-color-palette-blue-200: #1060ff;
+  --token-color-palette-blue-300: #0c56e9;
+  --token-color-palette-blue-400: #0046d1;
+  --token-color-foreground-high-contrast: #fff;
+  --token-color-focus-action-internal: #0c56e9;
+  --token-color-focus-action-external: #5990ff;
+
+  /* CSS properties */
   align-items: center;
-  background: none;
-  border: none;
-  border-radius: 4px;
-  color: var(--text-color-faded);
-  cursor: pointer;
+  background-color: var(--token-color-palette-blue-200);
+  border-color: var(--token-color-palette-blue-300);
+  border-radius: 5px;
+  border-style: solid;
+  border-width: var(--border-width);
+  color: var(--token-color-foreground-high-contrast);
   display: flex;
-  padding: 0 16px;
-  line-height: 32px;
+  font-family: system-ui, -apple-system, blinkmacsystemfont, 'Segoe UI', roboto,
+    helvetica, arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+    'Segoe UI Symbol';
```

🎨 (maybe 🚧 blocking?) This is interesting - we use `code-block` pretty heavily on marketing websites, and of course on the existing dot-ios which do _not_ use the system font stack... I think this might stand out a bit?

Definitely a hard balance to strike between dev-portal & marketing properties.

In other components, I think we rely on _not_ setting specific font stacks inline, and instead leaving `font-family` inherited, or using `var(--font-body)` which we have a bit more control over.

```suggestion
  font-family: var(--font-body);
```